lib/querystring.js

```diff
@@ -90,6 +90,9 @@ var hexTable = new Array(256);
 for (var i = 0; i < 256; ++i)
   hexTable[i] = '%' + ((i < 16 ? '0' : '') + i.toString(16)).toUpperCase();
 QueryString.escape = function(str) {
+  // replaces encodeURIComponent
+  // http://www.ecma-international.org/ecma-262/5.1/#sec-15.1.3.4
+  str = '' + str;
```

Comment:
I'm fine with that, it looks like if it's already a string, there's no performance penalty involved.